spec/factories/pacbio/plates.rb

```diff
@@ -3,7 +3,20 @@
     run { create(:pacbio_run) }
 
     factory :pacbio_plate_with_wells do
-      wells { create_list(:pacbio_well, 5)}
+      transient do
+        well_count { 1 }
+        well_factory { :pacbio_well }
+      end
+
+      wells { create_list(well_factory, well_count)}
+    end
+
+    trait :pooled do
+      transient do
+        pool_count { 1 }
+        # TODO: This needs sorting as we should be able to select the number of pools
```

Comment:
Is this a todo for now?

Reply:
Yes. It was a bit tricky to sort and wanted to timebox the story.